.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test pypairtree
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .
+          pip install pytest flake8
+      - name: Lint with flake8
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
+          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+          # exit-zero treats all errors as warnings.
+          flake8 pypairtree tests setup.py --count --exit-zero --max-line-length=99 --statistics
```

Comment:
Why do we run flake8 twice?

Reply:
That is a great question. I don't actually know why we run some of those things twice. It is probably unnecessary, but [here](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#using-the-python-starter-workflow) is the example we used.

Reply:
Just so that we have for reference of what some of the options are doing:
--count: Print the total number of errors.
--select: Specify the list of error codes you wish Flake8 to report.
  All E class violations are “errors” reported by pycodestyle.
  All F class violations are reported by pyflakes.
--show-source: Print the source code generating the error/warning in question.
--statistics: Count the number of occurrences of each error/warning code and print a report.